app/containers/SignIn/index.js

```diff
@@ -10,27 +10,29 @@ import {
   Platform,
   StyleSheet,
   View,
-} from 'react-native';
-import { TextField } from 'react-native-material-textfield';
-import { connect } from 'react-redux';
-import PropTypes from 'prop-types';
-import Button from 'react-native-button';
+  TextInput
+} from "react-native";
+import Icon from "react-native-vector-icons/Feather";
+import { TextField } from "react-native-material-textfield";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import Button from "react-native-button";
 
-import { signIn } from '../../actions/userActions';
-import { theme } from '../../styles';
-import ViewWrapper from '../../components/ViewWrapper';
-import LoadingIndicator from '../../components/LoadingIndicator';
+import { signIn } from "../../actions/userActions";
+import { theme } from "../../styles";
+import ViewWrapper from "../../components/ViewWrapper";
+import LoadingIndicator from "../../components/LoadingIndicator";
 
-const HEIGHT: number = Dimensions.get('window').height;
+const HEIGHT: number = Dimensions.get("window").height;
 
 class SignIn extends Component {
   static navigationOptions = {
-    header: null,
+    header: null
```

Comment:
```suggestion
    header: null,
```

Please use a comma at the end of the last property of each object literal (in all the places you have done the changes). The reason for this is, whenever you add a new property you will only have to change one line instead of two lines. This is really important for the code reviewing process.

Reply:
Oh, sure I'll fix all those problems, my linting software is probably not configured correctly